standard/operator.lua

```diff
@@ -0,0 +1,47 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Operator
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Operator = {}
+
+function Operator.add(a, b)
+	return a + b
+end
+
+function Operator.sub(a, b)
+	return a - b
+end
+
+function Operator.mul(a, b)
+	return a * b
+end
+
+function Operator.div(a, b)
+	return a / b
+end
+
+function Operator.pow(a, b)
+	return math.pow(a, b)
+end
+
+function Operator.item(item)
```

Comment:
Can also consider `Operator.at`, `Operator.field`.

Reply:
`.property`?

Reply:
i personally like `.property`

Reply:
sure why not